### DISCUSSION TITLE
does have second selection

### DISCUSSION THREAD

**Discussion by 1925381584** [helix-editor/helix#11309] [Ideas]
Is there a setting that allows me to select the first selection first. Then time is paused and then the second selection is selected. Of course, the meaning of selection seems to change here. Similar to the concept of the second selection in the meow plugin in emacs. This is especially useful when doing swaps of two selections or extending other tricks. I think this design helix is well worth taking a look at.

https://emacs-china.org/t/meow-grab/16071  
this is the Demo show

**Comment by pascalkuthe** [comment]
I am not sure what you mean your description is not clear to me. Linking a non-english page is not particularly helpful.

Helix has multi cursor/selection builtin and you can use selection rotation to facilitate swap operations. We want to add marks in the future to facilitate storing selections to a register and restoring them which may be similar to what you mean.

**Comment by 1925381584** [comment]
Thank you for your reply. I have a feeling that what you're about to do may very well be what I'm looking for. If choices could be manipulated more flexibly, it would be a new world of editing
